.eslintrc.js

```diff
@@ -23,6 +23,10 @@ module.exports = {
   },
   rules: {
     'ember/no-jquery': 'error',
+    'no-restricted-imports': ['error', {
```

Comment:
nice

Reply:
yeah, used that also for the upstream `ecsy-babylon`.